tests/conftest.py

```diff
@@ -19,9 +18,22 @@ def session_fixture():
 def client_fixture(session: Session):
     def get_session_override():
         return session
-
+    
     case_api.dependency_overrides[get_session] = get_session_override
 
     client = TestClient(case_api)
     yield client
     case_api.dependency_overrides.clear()
+
+@pytest.fixture(autouse=True)
+def auth_token(client):
+    # Send POST request with x-www-form-urlencoded data
+    response = client.post(
+        "/token",
+        data={"username": "johndoe", "password": "password"},
+        headers={"Content-Type": "application/x-www-form-urlencoded"}
+    )
```

Comment:
Can we please test the `/token` endpoint with data that doesn't contain a username, or password, or an incorrect username and password.